Task-RelayedData.md

```diff
@@ -0,0 +1,106 @@
+# SaltyRTC Relayed Data Task
+
+This task uses the end-to-end encrypted WebSocket connection set up by
+the SaltyRTC protocol to send user defined messages.
+
+# Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+# Terminology
+
+All terms from the [SaltyRTC protocol
+specification](./Protocol.md#terminology) are valid in this document.
+
+# Task Protocol Name
+
+The following protocol name SHALL be used for task negotiation:
+
+`v0.relayed-data.tasks.saltyrtc.org`
+
+# Task Data
+
+This task does not currently make use of the *data* field of the 'auth'
+messages described in the [SaltyRTC protocol
+specification](./Protocol.md#auth-message). The task data sent by the
+peers should be `Nil`.
+
+# Message Structure
+
+The same message structure as defined in the [SaltyRTC protocol
+specification](./Protocol.md#message-structure) SHALL be used.
+
+# Client-to-Client Messages
+
+Once the task has taken over, user data MAY be sent using 'data'
+messages.
```

Comment:
Can you clarify here that all other messages (other than 'close') defined by the main protocol are not allowed once the task kicks in?